src/main/ts/Patch.ts

```diff
@@ -373,29 +385,35 @@ type JQueryValProducer = (this: HTMLElement, index: number, value: JQueryValValu
  */
 const patchJqVal = (origFn: JQueryValFn): JQueryValFn =>
   function (this: JQuery<HTMLElement>, valueOrProducer?: JQueryValValue | JQueryValProducer): JQueryValValue | undefined | JQuery<HTMLElement> {
-    if (valueOrProducer === undefined) {
+    if (arguments.length === 0) {
       if (this.length >= 1) {
         return withTinymceInstance(this[0], (ed) => ed.getContent(), (elm) => origFn.call($(elm)));
       }
       // when no elements exist to query simply return undefined
       return undefined;
     } else {
       type ValSetterType = (valueOrProducer: JQueryValValue | JQueryValProducer) => JQuery<HTMLElement>;
+      // behave like original jQuery if argument is explicitely undefined
+      const fixedValueOrProducer = (
+        valueOrProducer === undefined
+          ? ''
+          : valueOrProducer
+      );
       this.each((i, el) => {
         withTinymceInstance(el, (ed) => {
-          const val = Type.isFunction(valueOrProducer) ? valueOrProducer.call(el, i, ed.getContent()) : valueOrProducer;
+          const val = Type.isFunction(fixedValueOrProducer) ? fixedValueOrProducer.call(el, i, ed.getContent()) : fixedValueOrProducer;
```

Comment:
The default value is only needed on this line. I would prefer you write it like this:
```suggestion
          const val = Type.isFunction(valueOrProducer) ? valueOrProducer.call(el, i, ed.getContent()) : valueOrProducer ?? '';
```

Reply:
Done